packages/renderer/src/lib/preferences/PreferencesRenderingItemFormat.svelte

```diff
@@ -21,6 +20,8 @@ export let setRecordValue = (_id: string, _value: string) => {};
 export let enableSlider = false;
 export let record: IConfigurationPropertyRecordedSchema;
 
+export let initialValue: any;
```

Comment:
it's not clear to me why type is any as it looks like you want to have an async function so it should at least say that it's a function

Reply:
The refactoring extracts the computation of the initial value out of the component and the initialValue is provided thus the any type. And because the method that now computes the value is async this is how it can be called in a Svelte loop according to the Svelte tutorial

Reply:
if initialValue is a function, then instead of `any` I would expect `() => Promise<any>` no ?

Reply:
or probably better `() => Promise<unknown>`